CHANGELOG.md

```diff
@@ -0,0 +1,141 @@
+# Changelog
+All notable changes to TouchFree software components are documented in this file.
+Change fragments are commonly categorised by the software component they relate to.
+
+Components include:
+- Service: The .NET TouchFree Service which processes hand data received from the Ultraleap Tracking Service into TouchFree input events
+- Tracking Service: The Ultraleap Tracking Service which manages connecting to camera hardware and running the hand tracking model
+- Installer: The TouchFree combined installer which installs all software components in a bundle
+- Settings: The settings UI opened from the tray icon
+- System Tray: The Windows tray icon
+- Overlay Application: The TouchFree Overlay Application launched from the tray icon that enables using TouchFree as a pointing device
+
+TouchFree web tooling changes are documented separately.
+See [here if looking at the source](./TF_Tooling_Web/CHANGELOG.md) otherwise [here on GitHub](https://github.com/ultraleap/TouchFree/blob/develop/TF_Tooling_Web/CHANGELOG.md).
+
+The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
+and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
+
+[docs-website]: https://docs.ultraleap.com/touchfree-user-manual/ "Ultraleap Docs"
```

Comment:
👀 Line missing (maybe because the space in the URL?)

Reply:
Replaced with some badges at the top (with discord and support email too)